homeassistant/components/plugwise/climate.py

```diff
@@ -122,49 +142,80 @@ def supported_features(self):
     def device_state_attributes(self):
         """Return the device specific state attributes."""
         attributes = {}
-        attributes["outdoor_temperature"] = self._outdoor_temperature
-        attributes["available_schemas"] = self._api.get_schema_names(
-            self._domain_objects
-        )
-        attributes["active_schema"] = self._active_schema
+        if self._outdoor_temperature:
+            attributes["outdoor_temperature"] = self._outdoor_temperature
+        attributes["available_schemas"] = self._schema_names
+        attributes["selected_schema"] = self._selected_schema
+        if self._illuminance:
+            attributes["illuminance"] = self._illuminance
```

Comment:
What is the illuminance measuring?

Reply:
Please explain what this measurement does. I'm thinking it should just be a sensor in a sensor platform.

Reply:
This is the illuminance sensor inside the Plugwise thermostat. A user requested its value to be added as an attribute.

Reply:
Sounds like it should be a separate sensor.

Reply:
Yes, this is planned for the next update of the Plugwise component.

Reply:
Good. Please remove it here then.

Reply:
OK